### PR TITLE
Create a Definition of Ready

### PR DESCRIPTION
**Pull request by nasadorian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,4 +38,11 @@ Green Light Health App helps businesses prevent spread of illness in their locat
 # Product Backlog:
 Find the [Product Backlog](https://github.com/jorgecotillo/green_light_health/projects/1)
 under github projects.
+
 # Definition of Ready:
+Our D.O.R. requires that the team be able to determine "what needs to be done" and the "amount of work" required to complete a given story.
+The below checklists should be complete to ensure a story is Ready.
```
Comment (nasadorian) [inline README.md]:
```suggestion
The below checklist should be complete to ensure a story is Ready.
```

File: README.md [hunk 2]
```diff
@@ -38,4 +38,11 @@ Green Light Health App helps businesses prevent spread of illness in their locat
...
+- [ ] Does the story capture and describe a reasonable and unit of work?
```
Comment (jorgecotillo) [inline README.md]: did you mean: "...a reasonable unit of work?" Based on what professor mentioned, everytime we see the word "and" it means a separate item, should we split this item into two?

File: README.md [hunk 3]
```diff
@@ -38,4 +38,11 @@ Green Light Health App helps businesses prevent spread of illness in their locat
...
+- [ ] Has the amount of work been estimated using an approved measurement?
```
Comment (jorgecotillo) [inline README.md]: Not sure what the rest of the team members think, but my preference would be to change the bullet points from questions to statements:

e.g. -> [ ] Work has been estimated using approved measurement (t-shirt size)

File: README.md (hunk 2 above)
Comment (nasadorian) [reply README.md]: Yup it's a typo, will address.

File: README.md (hunk 3 above)
Comment (nasadorian) [reply README.md]: I'm open to that format, sure

**Review by jorgecotillo — APPROVED** [review]
LGTM, if there are updates to make we can address them as a separate PR.

**Pull request merged**